### PR TITLE
Add functions to scenario tutorial and improve compare tutorial

### LINKED ISSUE #917: Documentation about gathering and displaying results of existing scenarios
The Core tutorial is actually pretty good for teaching the basics:
https://docs.taipy.io/en/latest/tutorials/fundamentals/2_scenario_management_overview/

The problem is as soon as you want something a bit out of the bounds of this tutorial, you're lost. We need more tutorials and demos on advanced core concepts. One of them is how do I gather and display the results of existing scenarios. A good example is in this app: 
https://github.com/Avaiga/demo-mlp-regularization
image

How do I gather the accuracies of my scenarios and display them in a bar chart?

### PR DESCRIPTION
**Pull request by FlorianJacta**
Resolves: #917 

image

image

image

image

image

image

### CODE REVIEW

File: docs/tutorials/scenario_management/6_scenario_comparison/index.md [hunk 1]
```diff
@@ -10,129 +10,182 @@ img: images/icon-code.svg
...
-            print(f"comparing result {current_res_i} with result {next_res_i}")
-            compare_result[current_res_i][next_res_i] = next_res - current_res
-            next_res_i += 1
-        current_res_i += 1
-    return compare_result
+def compare_revenue(*revenues):
+    scenario_names = [f"Scenario {i}" for i in range(len(revenues))]
+    comparisons = {"Scenarios": scenario_names,
+                   "Revenues": list(revenues)}
+    return pd.DataFrame(comparisons)
```
Comment (jrobinAV) [inline docs/tutorials/scenario_management/6_scenario_comparison/index.md:45]: Much better!

Thx

**Review by jrobinAV — APPROVED** [review]
It's better. Thx.
I would love @AlexandreSajus to review it. Last time you did not get how to compare scenarios when reading the tuto. With this new version, does it solve the pb?

File: docs/tutorials/scenario_management/6_scenario_comparison/index.md (hunk 1 above)
Comment (FlorianJacta) [reply docs/tutorials/scenario_management/6_scenario_comparison/index.md:45]: I think there is an API problem as I don't know which Scenario Ia m comparing in the compare function

Comment (jrobinAV) [reply docs/tutorials/scenario_management/6_scenario_comparison/index.md:45]: Indeed. That's a good point. 

The idea was to ask the user to define a comparison function as simple as possible. So we just provide the list of data to compare.

You are saying that you require more information than just the list of data. You are talking about the scenario, but the data node as well could be useful.

Can you create an issue about that?

Comment (FlorianJacta) [reply docs/tutorials/scenario_management/6_scenario_comparison/index.md:45]: https://github.com/Avaiga/taipy/issues/1233

Comment (jrobinAV) [reply docs/tutorials/scenario_management/6_scenario_comparison/index.md:45]: Thank you.

**Comment by AlexandreSajus** [comment]
> It's better. Thx. I would love @AlexandreSajus to review it. Last time you did not get how to compare scenarios when reading the tuto. With this new version, does it solve the pb?

Yes! This definitely solves my issue. The "some useful functions" part will definitely help. Good job!

**Review by AlexandreSajus — APPROVED** [review]
Good job!

**Pull request merged**